tools/ci_build/github/azure-pipelines/templates/c-api-cpu.yml

```diff
@@ -17,7 +17,7 @@ parameters:
 - name: DoEsrp
   displayName: Run code sign tasks? Must be true if you are doing an Onnx Runtime release.
   type: boolean
-  default: false
+  default: true
```

Comment:
Any reason to run esrp for nightly releases as well?

Reply:
Is it ok to publish an unsigned package to our 1st partners?

Reply:
It doesn't hurt to publish signed pkgs as such. I asked becoz they are just nightlies, not intended for production usage.

Reply:
I didn't however get how this prevented nightlies from getting published.